design/Alameda/design.md

```diff
@@ -0,0 +1,30 @@
+## What is Alameda
+
+Alameda is an intelligent resource orchestrator for Kubernetes, providing the features of autonomous balancing, scaling, and scheduling by using machine learning. Alameda learns the continuing changes of computing and I/O metrics from clusters, predicts the future demands for pods, and intelligently orchestrates underlying resources to fulfill the dynamic resource requests without manual configuration.
+
+For more details, please refer to https://github.com/containers-ai/Alameda
```

Comment:
the repo offers rather limited details

Reply:
Thanks for your review. We are working on it to our best.